vsphere/datadog_checks/vsphere/vsphere.py

```diff
@@ -153,17 +152,21 @@ def __init__(self, name, init_config, agentConfig, instances):
     def start_pool(self):
         self.log.info("Starting Thread Pool")
         self.pool_size = int(self.init_config.get('threads_count', DEFAULT_SIZE_POOL))
-
         self.pool = Pool(self.pool_size)
-        self.pool_started = True
```

Comment:
No need for it anymore: we start the pool at the beginning of the check, we stop it at the end